app/jobs/integrations/aggregator/credit_notes/create_job.rb

```diff
@@ -6,6 +6,8 @@ module CreditNotes
       class CreateJob < ApplicationJob
         queue_as 'integrations'
 
+        unique :until_executed
```

Comment:
I guess you should use the `on_conflict: :log` option, to avoid errors when calling duplicated `perform_later` 
```suggestion
        unique :until_executed, on_conflict: :log
```